dotrix_core/src/renderer.rs

```diff
@@ -315,6 +336,8 @@ pub enum Binding<'a> {
     Sampler(&'a str, Stage, &'a Sampler),
     /// Storage binding
     Storage(&'a str, Stage, &'a StorageBuffer),
+    /// Storage binding as Uniform
+    StorageAsUniform(&'a str, Stage, &'a StorageBuffer),
```

Comment:
I am curious but what was the use case that prompted this?

Reply:
As we discussed, it is to bind storage buffer as uniform buffer